virttest/utils_libvirt/libvirt_vfio.py

```
@@ -12,6 +12,21 @@
+def get_vfio_pci(pci_id):
+    """
+    Get the driver name for a PCI device
+
+    :param pci_id: The id of pci device
+    :return: The driver name
+    """
+    cmd = (
+        "readlink -f /sys/bus/pci/devices/%s/driver "
+        "| awk -F '/' '{print $NF}'" % pci_id
+    )
+    output = process.run(cmd, shell=True, verbose=True).stdout_text.strip()
```

Ruff (0.14.1)

Function call with `shell=True` parameter identified, security issue

(S604)